examples/rust/pipe-ping-pong.rs

```diff
@@ -294,8 +258,8 @@ impl PipeClient {
 // Drop implementation for the pipe client.
 impl Drop for PipeClient {
     fn drop(&mut self) {
-        self.close(self.pipeqd_rx);
-        self.close(self.pipeqd_tx);
+        self.close(self.pipeqd);
+        self.close(self.pipeqd);
```

Comment:
pipeqd is closed twice here. Is this desirable?

Reply:
No, fixed.